### DISCUSSION TITLE
The particle kinetic energy density

### DISCUSSION THREAD

**Discussion by shuipingqiezi** [SmileiPIC/Smilei#738] [Q&A]
The "Ukin" in my scalars.txt is 5.2175939829e+04, I cannot convert it to the initial gamma=200.

**Comment by mccoys** [comment]
Please do not post comments here. Use the "discussions". And please also ask a question instead of a remark.

**Comment by mccoys** [comment]
Note that ukin is the total kinetic energy in the simulation. I guess your gamma is for 1 particle
